src/Sulu/Bundle/AdminBundle/Resources/js/views/Form/toolbarActions/DeleteToolbarAction.js

```diff
@@ -4,22 +4,34 @@ import {action, computed, observable} from 'mobx';
 import jexl from 'jexl';
 import log from 'loglevel';
 import Dialog from '../../../components/Dialog';
-import {translate} from '../../../utils/Translator';
+import DeleteDependantsDialog from '../../../containers/DeleteDependantsDialog';
+import DeleteReferencedResourceDialog from '../../../containers/DeleteReferencedResourceDialog';
+import {translate} from '../../../utils';
 import {ResourceFormStore} from '../../../containers/Form';
 import Router from '../../../services/Router';
 import ResourceStore from '../../../stores/ResourceStore';
 import Form from '../Form';
 import AbstractFormToolbarAction from './AbstractFormToolbarAction';
+import type {Resource} from '../../../types';
 
 export default class DeleteToolbarAction extends AbstractFormToolbarAction {
     @observable showDialog: boolean = false;
-    @observable showLinkedDialog: boolean = false;
-    @observable referencingItems: Array<Object> = [];
+    @observable showDeleteReferencedResourcesDialog: boolean = false;
```

Comment:
like above, not sure if `showDeleteReferencedResourcesDialog` and `referencingResourcesData` is needed